helm.tf

```diff
@@ -0,0 +1,28 @@
+# helm.tf
+
+provider "helm" {
+  kubernetes {
+    host                   = azurerm_kubernetes_cluster.default.kube_config[0].host
+    client_certificate     = base64decode(azurerm_kubernetes_cluster.default.kube_config[0].client_certificate)
+    client_key             = base64decode(azurerm_kubernetes_cluster.default.kube_config[0].client_key)
+    cluster_ca_certificate = base64decode(azurerm_kubernetes_cluster.default.kube_config[0].cluster_ca_certificate)
+  }
+}
+
+resource "helm_release" "cilium" {
+  name       = "cilium"
+  repository = "https://helm.cilium.io/"
+  chart      = "cilium"
+  version    = "1.12.2" # specify the version you want to use
```

Comment:
This is a very outdated version of Cilium, let's use the most up to date version